pandas/core/config_init.py

```diff
@@ -277,7 +279,7 @@ def use_numexpr_cb(key):
     Valid values: False,True
 """
 
-style_backup = dict()
+style_backup = dict()  # type: Dict
```

Comment:
Is this variable still needed? I searched the codebase for `style_backup`, and it doesn't appear anywhere else. It might be some vestigial code.

Reply:
yeah looks like you can remove it